### ISSUE TITLE
Request: Add option to define the "required output" in belts instead of items

### ISSUE THREAD

**Issue by USSX-Hares** [KirkMcDonald/kirkmcdonald.github.io#244]
Usually, you when desiging a factory module, you want that module to produce, 6/9/12/etc belts of output.
Of course, you can multiply number of belts by 45 (for items/sec) or 2700 (for items/min), but it's painful to do so every time.

**Comment by Bonnev** [comment]
I agree. It would also be great to be able to choose the type of belt (yellow, red, blue) for earlier in the game.

**Comment by USSX-Hares** [comment]
@Bonnev, you can select type of belt in settings tab.

**Comment by Bonnev** [comment]
True, that can be used! I'm just pointing out that it's not always 45.
